src/analyzer/protocol/smb/smb1-com-query-information.pac

```diff
@@ -31,7 +31,7 @@ type SMB1_query_information_request(header: SMB_Header) = record {
 type SMB1_query_information_response(header: SMB_Header) = record {
 	word_count      : uint8;
 	file_attribs    : uint16;
-	last_write_time : SMB_time;
+	last_write_time : uint32;
```

Comment:
I've found the following refs and it seems uint32 for the UTIME format is good.

https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-cifs/cfa18c97-61c7-422e-ab64-cd3849864dfd

https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-cifs/94ef49b8-648c-47f1-9c64-def4ad0a63a0

The docs of UTIME mention little endian explicitly. I'm assuming that fields are parsed littleendian due to the outer `SMB_Message_Response` record having `&byteorder`.